stix_shifter_modules/qradar/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -465,7 +465,9 @@
       "key": "domain-name.value",
       "object": "domain2",
       "transformer": "ToDomainName"
-    },
+    }
+  ],
+  "qradardomain": [
```

Comment:
You should remove the entry that's currently under `dnsdomainname` a few lines down. We don't want to write duplicated data in multiple properties. In fact, if dnsdomainname is truly a domain, it shouldn't be writing the value to the url object.